workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,20 +51,35 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
+        {
+            GetClientLogFilename();
+            Launch("Launch standalone client", "local worker launch UnityClient default", "*unityclient.log");
+        }
+
+        public static void LaunchLocalDeployment()
         {
             BuildConfig();
+            Launch("Launch SpatialOS locally", "local launch", "spatial_*.log");
```

Comment:
Do we not already have tools for this?

Reply:
No idea - where can I take a look at the tools?